create_book.rb

```diff
@@ -8,6 +9,9 @@ def create_book
   author = gets.chomp
   book = Book.new(title, author)
   @books.push(book)
+  save_book(title, author)
```

Comment:
- [ ] you are saving to the json file each time the book gets created, the issue with this is that after adding the book and then showing the list of books the added book will appear twice. you can solve this by updating all the json files when the user is closing the app.
![afbeelding](https://user-images.githubusercontent.com/39240820/218082491-cae163e1-f85a-4142-9a07-01af40b75dc1.png)